### PR TITLE
Flakified the remaining code

### PR DESCRIPTION
**Pull request by almarklein**
issues: related to #2078

This makes the whole bokeh codebase compliant with pyflakes. I increased the limit for long lines to 160 to avoid having to change many more lines.

Also, all files called `__init__.py` are not style-checked: many of these contain imports that are not used. Therefore its probably better to avoid putting actual code in any `__init__.py`.

### CODE REVIEW

File: bokeh/templates.py [hunk 1]
```diff
@@ -64,7 +64,10 @@
     plot_div (str) : typically the output of PLOT_DIV
 
 Attributes:
-    AUTOLOAD: This template is for creating a sidecar JS file that will automatically and asynchronously load BokehJS (if necessary) and then replaces the ``script`` tag that loads it (with appropriate attributes) with a rendered plot.
+    AUTOLOAD: This template is for creating a sidecar JS file that will
```
Comment (bryevdv) [inline bokeh/templates.py]: Note sure Sphinx will render this correctly. Suggest:
```
    AUTOLOAD: This template is for creating an "autoload" JS script.
        The script automatically and asynchronously loads BokehJS (if necessary) 
        and then replaces the ``script`` tag that loads it (with appropriate attributes) with 
        a rendered plot.
```

File: bokeh/transforms/image_downsample.py [hunk 2]
```diff
@@ -11,20 +11,21 @@
...
-                    'dw' : [10],
-                    'dh' : [10],
-                   }
-  return ServerDataSource(**kwargs)
+    kwargs['transform'] = {'resample':'heatmap',
+                           'global_x_range' : [0, 10],
+                           'global_y_range' : [0, 10],
+                           'global_offset_x' : [0],
+                           'global_offset_y' : [0],
+                           'type' : 'ndarray'
```
Comment (bryevdv) [inline bokeh/transforms/image_downsample.py]: Trailing comma recommended on last lines. FWIW I really hate the aligned brace style. I'd personally rather see
```
kwargs['transform'] = {
    'resample':'heatmap',
    'global_x_range' : [0, 10],
    'global_y_range' : [0, 10],
    'global_offset_x' : [0],
    'global_offset_y' : [0],
    'type' : 'ndarray',
}
```
or
```
kwargs['transform'] = dict(
    resample='heatmap',
    global_x_range=[0, 10],
    global_y_range=[0, 10],
    global_offset_x=[0],
    global_offset_y=[0],
    type'='ndarray',
)
```

File: bokeh/templates.py (hunk 1 above)
Comment (almarklein) [inline bokeh/templates.py]: I'll check and adopt your changes.

File: bokeh/simpleapp.py [hunk 3]
```diff
@@ -8,6 +6,8 @@
 from .properties import String, Instance, Dict, Any, Either
 from .plotting import curdoc, cursession, push, show as pshow
 
+(HBox, VBox, VBoxForm, SimpleApp, PlotObject)
+(String, Instance, Dict, Any, Either)
```
Comment (almarklein) [inline bokeh/simpleapp.py]: I assume these ought to be here?

File: bokeh/tests/test_properties.py [hunk 4]
```diff
@@ -315,25 +317,6 @@ class Foo(HasProps):
...
-        with self.assertRaises(ValueError):
-            f.pat = [2, 4.2]
-        with self.assertRaises(ValueError):
-            f.pat = [2, "a"]
-
-    def test_list(self):
-        class Foo(HasProps):
-            pat = DashPattern
-        f = Foo()
-
```
Comment (almarklein) [inline bokeh/tests/test_properties.py]: This function was an exact duplicate.

File: bokeh/transforms/image_downsample.py (hunk 2 above)
Comment (almarklein) [inline bokeh/transforms/image_downsample.py]: hehe, I also prefer that, but I suppose I'm now trained to use the style that pyflakes wants me to use. I shall change it.

File: bokeh/templates.py (hunk 1 above)
Comment (almarklein) [inline bokeh/templates.py]: Sphinx output is correct in this way.

**Comment by almarklein** [comment]
Travis fails because some example use itertools, which they previously imported from `bokeh.plotting` (via a star import). I propose to update the examples to import itertools directly?

**Comment by almarklein** [comment]
Ready from my end.

**Comment by almarklein** [comment]
ping

**Pull request merged**